.github/workflows/watch.yml

```diff
@@ -0,0 +1,36 @@
+name: Watch
+
+on:
+  schedule:
+    - cron: '0 2,8,14,20 * * *'
+
+jobs:
+  docker:
+    name: Push docker image
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: master
+
+      - name: Update Dockerfile with latest version
+        id: fetch_version
+        run: |
+          last=$(curl -s https://packagist.org/packages/phpstan/phpstan.json|jq '[.package.versions[]|select(.version|test("^\\d+\\.\\d+\\.\\d+$"))|.version]|max_by(.|[splits("[.]")]|map(tonumber))')
+          last=$(echo $last | tr -d '"')
+          echo "Last PHPStan version is $last"
+          echo ::set-output name=last::$last
+
+          sed -i -re "s/require phpstan\/phpstan [^\\]+/require phpstan\/phpstan $last /" Dockerfile
+          cat Dockerfile
+
+      - name: Docker login
+        run: echo '${{ secrets.DOCKER_PASSWORD }}' | docker login --username ${{ secrets.DOCKER_USERNAME }} --password-stdin
```

Comment:
@OskarStark 

Are these secrets defined?

Reply:
not yet